lib/components/ms-toggle/MsBooleanToogle.vue

```diff
@@ -0,0 +1,75 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="ms-boolean-toggle">
+    <ion-text
+      type="button"
+      class="button-view button-medium"
+      :class="{ 'button-disabled': modelValue === Answer.No }"
+      :disabled="modelValue === Answer.Yes"
```

Comment:
`disabled` means that you cannot interact with the element, it should not be used to change appearances.

Reply:
As discussed, `disabled` is used to prevent the user from clicking on something that is already selected.